### PR TITLE
Fix Pulse kafka-appender arg

### PR DESCRIPTION
**Pull request by Niveditha-phData**
This will fix the "Bad arguments for option 'kafka-properties' " error on log-collector kafka.

### CODE REVIEW

File: bin/control.sh [hunk 1]
```diff
@@ -42,6 +42,9 @@ export AKKA_CONF=${AKKA_CONF:-"application.conf"}
 export ALERT_ENGINE_MAX_HEAP=${ALERT_ENGINE_MAX_HEAP:-"2G"}
 export COLLECTION_ROLLER_MAX_HEAP=${COLLECTION_ROLLER_MAX_HEAP:-"2G"}
 export LOG_COLLECTOR_MAX_HEAP=${LOG_COLLECTOR_MAX_HEAP:-"8G"}
+export KAFKA_PROPS=${KAFKA_PROPS:-"./cloudera-integration/csd/aux/kafka.properties"}
```
Comment (afoerster) [inline bin/control.sh]: This is correct when you run it locally but will fail when deployed on a cluster. The correct default value on a cluster should be just `kafka.properties`

File: cloudera-integration/csd/aux/kafka.properties [hunk 2]
```diff
@@ -0,0 +1,9 @@
+bootstrap.servers=master1.valhalla.phdata.io:9093,master2.valhalla.phdata.io:9093,master3.valhalla.phdata.io:9093
```
Comment (afoerster) [inline cloudera-integration/csd/aux/kafka.properties]: These are environmental configs so we shouldn't commit them into the project. Just leave this and the trust store properties out, then we need to document them.

File: cloudera-integration/csd/descriptor/service.sdl [hunk 3]
```diff
@@ -160,6 +160,10 @@
         {
           "filename" : "application.conf",
           "sourceFilename" : "aux/application.conf"
+        },
+        {
+          "filename" : "kafka.properties",
```
Comment (afoerster) [inline cloudera-integration/csd/descriptor/service.sdl]: This is the file that ends up in `KAFKA_PROPS`

File: cloudera-integration/csd/aux/kafka.properties (hunk 2 above)
Comment (Niveditha-phData) [reply cloudera-integration/csd/aux/kafka.properties]: Should I leave `kafka.properties` as empty file?

File: bin/control.sh [hunk 4]
```diff
@@ -42,6 +42,9 @@ export AKKA_CONF=${AKKA_CONF:-"application.conf"}
 export ALERT_ENGINE_MAX_HEAP=${ALERT_ENGINE_MAX_HEAP:-"2G"}
 export COLLECTION_ROLLER_MAX_HEAP=${COLLECTION_ROLLER_MAX_HEAP:-"2G"}
 export LOG_COLLECTOR_MAX_HEAP=${LOG_COLLECTOR_MAX_HEAP:-"8G"}
+export KAFKA_PROPS=${KAFKA_PROPS:-"kafka.properties"}
+export KAFKA_OPTS="-Djava.security.auth.login.config=${CONF_DIR}/jaas.conf"
+export KAFKA_TOPIC=${KAFKA_TOPIC:-"pulse-kafka-topic"}
```
Comment (afoerster) [inline bin/control.sh]: I think we should leave this empty. Having a default makes it easier to not add the property, it fails, and the user wonders where this topic name came from

Comment (afoerster) [reply bin/control.sh]: Is there somewhere in the service.sdl where the KAFKA_TOPIC is set?

Comment (Niveditha-phData) [reply bin/control.sh]: > Is there somewhere in the service.sdl where the KAFKA_TOPIC is set?
Yea. It is set in service.sdl

File: bin/env.sh.template [hunk 5]
```diff
@@ -22,6 +22,6 @@ export ALERT_ENGINE_MAX_HEAP=2048m
 export COLLECTION_ROLLER_MAX_HEAP=2048m
 export LOG_COLLECTOR_MAX_HEAP=4096m
 export KAFKA_OPTS="-Djava.security.auth.login.config=$CONF_DIR/jaas.conf"
-export KAFKA_PROPS="$CONF_DIR/kafka.properties"
+export KAFKA_PROPS=kafka.properties
```
Comment (afoerster) [inline bin/env.sh.template]: Can you just get rid if this variable and put it directly in the command?

Comment (afoerster) [reply bin/env.sh.template]: Oh my bad, this is the env.sh file. This is good.

**Pull request merged**